<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>matrix</class>
 <widget class="QMainWindow" name="matrix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>matrix</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#111;
color:white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_6">
      <property name="styleSheet">
       <string notr="true">font-size:24px;
font-weight:1000;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item alignment="Qt::AlignHCenter|Qt::AlignTop">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>برنامه ی ماتریس </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color:rgb(0, 170, 0);
	color:white;
	padding:5px;
 	border: 1px solid rgb(0, 170, 0);
 	border-radius: 4px;
	font-size:16px;
}</string>
         </property>
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  appearance: button;
  background-color: #1652F0;
  border: 1px solid #1652F0;
  border-radius: 4px;
  box-sizing: border-box;
  color: #FFFFFF;
  cursor: pointer;
  font-family: Graphik,-apple-system,system-ui,&quot;Segoe UI&quot;,Roboto,Oxygen,Ubuntu,Cantarell,&quot;Fira Sans&quot;,&quot;Droid Sans&quot;,&quot;Helvetica Neue&quot;,sans-serif;
  font-size: 14px;
  line-height: 1.15;
  overflow: visible;
  padding: 12px 16px;
  position: relative;
  text-align: center;
  text-transform: none;
  transition: all 80ms ease-in-out;
  user-select: none;
  -webkit-user-select: none;
  touch-action: manipulation;
  width: fit-content;
}

QPushButton::disabled {
  opacity: .5;
}

QPushButton::focus {
  outline: 0;
}

QPushButton::hover {
  background-color: #0A46E4;
  border-color: #0A46E4;
}

QPushButton::active {
  background-color: #0039D7;
  border-color: #0039D7;
}</string>
         </property>
         <property name="text">
          <string>ماتریس رندوم</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="styleSheet">
          <string notr="true">QFrame{
	background-color:#222;
	color:white;
	border:1px solid #333;
	border-radius: 4px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item alignment="Qt::AlignRight">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font-size:20px;
font-weight:700;
border:none;</string>
            </property>
            <property name="text">
             <string>تعداد ستون</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="insert_j">
            <property name="styleSheet">
             <string notr="true">border:1px solid #333;
border-radius: 4px;
color:white;
padding:5px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="styleSheet">
          <string notr="true">QFrame{
	background-color:#222;
	color:white;
	border:1px solid #333;
	border-radius: 4px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">font-size:20px;
font-weight:700;
border:none;</string>
            </property>
            <property name="text">
             <string>تعداد سطر</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="insert_i">
            <property name="styleSheet">
             <string notr="true">border:1px solid #333;
border-radius: 4px;
color:white;
padding:5px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="styleSheet">
       <string notr="true">background-color:#222;
color:white;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font-weight:700;</string>
         </property>
         <property name="text">
          <string>سطر را وارد کنید: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="rowVal">
            <property name="styleSheet">
             <string notr="true">border:1px solid #333;
border-radius: 4px;
color:white;
padding:5px;
background-color:#111;</string>
            </property>
            <property name="placeholderText">
             <string>1 2 3 4 5 ...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color:rgb(0, 170, 0);
	color:white;
	padding:5px;
 	border: 1px solid rgb(0, 170, 0);
 	border-radius: 4px;
}</string>
            </property>
            <property name="text">
             <string>افزودن ماتریس</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_7">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="changiToj">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  appearance: button;
  background-color: #1652F0;
  border: 1px solid #1652F0;
  border-radius: 4px;
  box-sizing: border-box;
  color: #FFFFFF;
  cursor: pointer;
  font-family: Graphik,-apple-system,system-ui,&quot;Segoe UI&quot;,Roboto,Oxygen,Ubuntu,Cantarell,&quot;Fira Sans&quot;,&quot;Droid Sans&quot;,&quot;Helvetica Neue&quot;,sans-serif;
  font-size: 14px;
  line-height: 1.15;
  overflow: visible;
  padding: 12px 16px;
  position: relative;
  text-align: center;
  text-transform: none;
  transition: all 80ms ease-in-out;
  user-select: none;
  -webkit-user-select: none;
  touch-action: manipulation;
  width: fit-content;
}

QPushButton::disabled {
  opacity: .5;
}

QPushButton::focus {
  outline: 0;
}

QPushButton::hover {
  background-color: #0A46E4;
  border-color: #0A46E4;
}

QPushButton::active {
  background-color: #0039D7;
  border-color: #0039D7;
}</string>
         </property>
         <property name="text">
          <string>Transpose</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="toepliz_check">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  appearance: button;
  background-color: #1652F0;
  border: 1px solid #1652F0;
  border-radius: 4px;
  box-sizing: border-box;
  color: #FFFFFF;
  cursor: pointer;
  font-family: Graphik,-apple-system,system-ui,&quot;Segoe UI&quot;,Roboto,Oxygen,Ubuntu,Cantarell,&quot;Fira Sans&quot;,&quot;Droid Sans&quot;,&quot;Helvetica Neue&quot;,sans-serif;
  font-size: 14px;
  line-height: 1.15;
  overflow: visible;
  padding: 12px 16px;
  position: relative;
  text-align: center;
  text-transform: none;
  transition: all 80ms ease-in-out;
  user-select: none;
  -webkit-user-select: none;
  touch-action: manipulation;
  width: fit-content;
}

QPushButton::disabled {
  opacity: .5;
}

QPushButton::focus {
  outline: 0;
}

QPushButton::hover {
  background-color: #0A46E4;
  border-color: #0A46E4;
}

QPushButton::active {
  background-color: #0039D7;
  border-color: #0039D7;
}</string>
         </property>
         <property name="text">
          <string>Toepliz</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_4">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  appearance: button;
  background-color: #1652F0;
  border: 1px solid #1652F0;
  border-radius: 4px;
  box-sizing: border-box;
  color: #FFFFFF;
  font-family: Graphik,-apple-system,system-ui,&quot;Segoe UI&quot;,Roboto,Oxygen,Ubuntu,Cantarell,&quot;Fira Sans&quot;,&quot;Droid Sans&quot;,&quot;Helvetica Neue&quot;,sans-serif;
  font-size: 14px;
  line-height: 1.15;
  overflow: visible;
  padding: 12px 16px;
  position: relative;
  text-align: center;
  text-transform: none;
  transition: all 80ms ease-in-out;
  user-select: none;
  -webkit-user-select: none;
  touch-action: manipulation;
  width: fit-content;
}

QPushButton::disabled {
  opacity: .5;
}

QPushButton::focus {
  outline: 0;
}

QPushButton::hover {
  background-color: #0A46E4;
  border-color: #0A46E4;
}

QPushButton::active {
  background-color: #0039D7;
  border-color: #0039D7;
}</string>
         </property>
         <property name="text">
          <string>Center Check</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_5">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  appearance: button;
  background-color: #1652F0;
  border: 1px solid #1652F0;
  border-radius: 4px;
  box-sizing: border-box;
  color: #FFFFFF;
  font-family: Graphik,-apple-system,system-ui,&quot;Segoe UI&quot;,Roboto,Oxygen,Ubuntu,Cantarell,&quot;Fira Sans&quot;,&quot;Droid Sans&quot;,&quot;Helvetica Neue&quot;,sans-serif;
  font-size: 14px;
  line-height: 1.15;
  overflow: visible;
  padding: 12px 16px;
  position: relative;
  text-align: center;
  text-transform: none;
  transition: all 80ms ease-in-out;
  user-select: none;
  -webkit-user-select: none;
  touch-action: manipulation;
  width: fit-content;
}

QPushButton:disabled {
  opacity: .5;
}

QPushButton::focus {
  outline: 0;
}

QPushButton::hover {
  background-color: #0A46E4;
  border-color: #0A46E4;
}

QPushButton:active {
  background-color: #0039D7;
  border-color: #0039D7;
}</string>
         </property>
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  appearance: button;
  background-color: #1652F0;
  border: 1px solid #1652F0;
  border-radius: 4px;
  box-sizing: border-box;
  color: #FFFFFF;
  font-family: Graphik,-apple-system,system-ui,&quot;Segoe UI&quot;,Roboto,Oxygen,Ubuntu,Cantarell,&quot;Fira Sans&quot;,&quot;Droid Sans&quot;,&quot;Helvetica Neue&quot;,sans-serif;
  font-size: 14px;
  line-height: 1.15;
  overflow: visible;
  padding: 12px 16px;
  position: relative;
  text-align: center;
  text-transform: none;
  transition: all 80ms ease-in-out;
  user-select: none;
  -webkit-user-select: none;
  touch-action: manipulation;
  width: fit-content;
}

QPushButton::disabled {
  opacity: .5;
}

QPushButton::focus {
  outline: 0;
}

QPushButton::hover {
  background-color: #0A46E4;
  border-color: #0A46E4;
}

QPushButton::active {
  background-color: #0039D7;
  border-color: #0039D7;
}</string>
         </property>
         <property name="text">
          <string>Print matrix</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_3">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  appearance: button;
  background-color: #1652F0;
  border: 1px solid #1652F0;
  border-radius: 4px;
  box-sizing: border-box;
  color: #FFFFFF;
  font-family: Graphik,-apple-system,system-ui,&quot;Segoe UI&quot;,Roboto,Oxygen,Ubuntu,Cantarell,&quot;Fira Sans&quot;,&quot;Droid Sans&quot;,&quot;Helvetica Neue&quot;,sans-serif;
  font-size: 14px;
  line-height: 1.15;
  overflow: visible;
  padding: 12px 16px;
  position: relative;
  text-align: center;
  text-transform: none;
  transition: all 80ms ease-in-out;
  user-select: none;
  -webkit-user-select: none;
  touch-action: manipulation;
  width: fit-content;
}

QPushButton::disabled {
  opacity: .5;
}

QPushButton::focus {
  outline: 0;
}

QPushButton::hover {
  background-color: #0A46E4;
  border-color: #0A46E4;
}

QPushButton::active {
  background-color: #0039D7;
  border-color: #0039D7;
}</string>
         </property>
         <property name="text">
          <string>Clear Terminal</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_8">
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color:#222;
	color:white;
	border:1px solid #333;
	border-radius: 4px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item alignment="Qt::AlignTop">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font-size:20px;
border:none;</string>
         </property>
         <property name="text">
          <string>خروجی</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="console">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>console </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_8">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color:rgb(0, 170, 0);
	color:white;
	padding:5px;
 	border: 1px solid rgb(0, 170, 0);
 	border-radius: 4px;
}</string>
         </property>
         <property name="text">
          <string>ذخیره ی ترمینال</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport_Terminal">
   <property name="text">
    <string>Export Terminal</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
